src/App/Start/useLocalUrls.tsx

```diff
@@ -3,11 +3,19 @@ import { detectLocalIP } from "../../lib/detectLocalIP.js"
 const localIp = detectLocalIP()
 
 export function useLocalUrls(port?: number) {
```

Comment:
I know it is from previously, but lets rename it from use to something else so it would not look like a hook?

Reply:
renamed 👍